Automattic/sensei
# Update quiz navigation and title area in learning mode

Resolves https://github.com/Automattic/sensei/issues/7072

## Proposed Changes
* Updated the quiz template to modify the title area
* Removed quiz progress as per design
* Introduced compat file for quiz to add quiz specific designs in this file separated from lesson, to prevent everything being on main compat file
* Updated spacings and stylings to match the design
* Changed the arrow before the 'Back to lesson' button from '<' character to arrow icon

## Testing Instructions


1. Checkout this branch of Course theme https://github.com/Automattic/themes/pull/7312
2. Checkout this branch
3. Activate course theme
4. Create a Course with a lesson, add a quiz to that lesson
5. Enable learning mode
6. Take the quiz as a student from frontend
7. Make sure the Back to Lesson button and Quiz title matches the design
8. Check for all variations
9. Now switch to other top themes to sensei and make sure the design matches
10. Check each of the above for mobile as well

### Screenshots
<img width="1160" alt="Screenshot 2023-08-14 at 4 26 16 PM" src="https://github.com/Automattic/sensei/assets/6820724/1991acc2-16e0-4dcc-b3aa-8bc525c56fbb">
<img width="479" alt="Screenshot 2023-08-14 at 4 27 03 PM" src="https://github.com/Automattic/sensei/assets/6820724/73af892a-81e8-4ab9-99d3-4073aaeef09d">

## Pre-Merge Checklist
